ci-operator/step-registry/interop-tests/cnv-tests-e2e-deploy/interop-tests-cnv-tests-e2e-deploy-commands.sh

```
@@ -78,24 +59,18 @@ make deploy_test || exit_code=$?
- # Send junit files to shared dir for Data Router Reporter step
-cp "${ARTIFACT_DIR}"/*.xml "${SHARED_DIR}"
-
-
 if [ "${exit_code:-0}" -ne 0 ]; then
-    echo "deploy_test failed with exit code $exit_code"
+    # deploy_test failed; exit status is propagated below (xtrace is off in this block).
     if [[ -n "${CNV_WAIT_FOR_LIVE_DEBUG:-}" ]]; then
-        echo "Waiting to allow interactive debugging"
-        if ((CNV_WAIT_FOR_LIVE_DEBUG > 0)); then
+        # Hold for live debugging: positive CNV_WAIT_FOR_LIVE_DEBUG = bounded sleep; otherwise sleep inf.
+        if ((${CNV_WAIT_FOR_LIVE_DEBUG} > 0)); then
```

Shellcheck (0.11.0)

[style] $/${} is unnecessary on arithmetic variables.

(SC2004)